ci-operator/config/openshift/cloud-provider-gcp/openshift-cloud-provider-gcp-main.yaml

```diff
@@ -123,17 +123,16 @@ tests:
   container:
     from: commitchecker
   optional: true
+- as: verify-vendor
+  commands: tools/verify-vendor.sh
+  container:
+    from: src
+  optional: true
```

Comment:
Why optional? Pretty sure that we'll need to make this run on all jobs to ensure that the code is properly placed.

Reply:
I'm pretty sure running `make-jobs` added this in, I didn't notice this when merging on the rebase

Reply:
Sorry, double checked that and perhaps not :) It's set to false now